### PR TITLE
Documentation for manually sending out validations

### LINKED ISSUE #1: How to manually send out validations
If a customer chooses to send out a CPR validation not using the Penneo system to generate the emails, if someone tries to resend the validation from the web application, the result will be that the validation can't be send.

Investigate whether the email text and email subject can be set after the validation has been activated using the "send" method and the link has been extracted.

If this is possible, update the documentation to specify this as the default procedure when you want to handle email sending yourself.

### PR DESCRIPTION
**Pull request by ahmadnazir**
Fixes #1

### CODE REVIEW

File: docs/validation.md [hunk 1]
```diff
@@ -46,10 +46,20 @@ $myValidation->setName('John Doe');
 Validation::persist($myValidation);
 
 // Activate the validation object
+// As the email content isn't defined, nothing will be sent
 $myValidation->send();
 
 // Retrieve the validation link
 $myLink = $myValidation->getLink();
+
+// Define the content of the email if you would like to send emails through Penneo as well
```
Comment (janflora) [inline docs/validation.md]: Could you elaborate on this to make it very clear that this applies if you want to **re-send** the signing request **through** Penneo at a later point.
I'm a pain in the ass, I know ;-)

Comment (ahmadnazir) [inline docs/validation.md]: Will do .. :+1:

Comment (ahmadnazir) [inline docs/validation.md]: But you mean validation request, not signing request..

**Comment by janflora** [comment]
Merged

**Pull request closed without merging**